build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
         }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:4.0.0'
+        classpath 'com.android.tools.build:gradle:4.1.1'
```

Comment:
do we really want to update this in the feature branch? i would prefer to change this in its own branch, to (if something is not working anymore) we can decide to not yet update.

Reply:
no idea where this came from, removed it